### PR TITLE
fix: add float conversion for default timeout. add upcloud api config…

### PR DESCRIPTION
**Pull request by Trdls**
… as env variables, in case user doesn't use env variables

### CODE REVIEW

File: inventory/upcloud.py [hunk 1]
```diff
@@ -232,11 +232,18 @@ def read_api_credentials(config):
...
-        default_timeout = config.get('upcloud', 'default_timeout') or None
+        default_timeout = None
+    else:
+        default_timeout = float(default_timeout)
     manager = upcloud_api.CloudManager(username, password, default_timeout)
 
+    # In case user doesn't use env variables, we might need them in modules
+    os.environ['UPCLOUD_API_USER'] = username
+    os.environ['UPCLOUD_API_PASSWD'] = password
+    os.environ['UPCLOUD_API_TIMEOUT'] = default_timeout
```
Comment (Darep) [inline inventory/upcloud.py]: Does this work? 🤔 Not familiar so not sure if it's possible to set env vars like this, sometimes they are readonly 🤔

Comment (Trdls) [reply inventory/upcloud.py]: This could be one of those "Works on my machine" ™️ type of deals, so not 100% sure.

File: modules/upcloud.py [hunk 2]
```diff
@@ -315,16 +315,17 @@ def main():
     api_user = module.params.get('api_user') or os.getenv('UPCLOUD_API_USER')
     api_passwd = module.params.get('api_passwd') or os.getenv('UPCLOUD_API_PASSWD')
     default_timeout = os.getenv('UPCLOUD_API_TIMEOUT')
+
     if not default_timeout:
-        default_timeout = module.params.get('upcloud', 'default_timeout') or None
```
Comment (Darep) [inline modules/upcloud.py:319]: Why was this removed? Should the same default_timeout logic be here too (from above)? I'm not very familiar with the call structure here 😅

File: inventory/upcloud.py (hunk 1 above)
Comment (Darep) [reply inventory/upcloud.py]: If it works on your machine and doesn't crash then LGTM 😄

File: modules/upcloud.py (hunk 2 above)
Comment (Trdls) [reply modules/upcloud.py:319]: These are ansible module params, since the user might use their upcloud account as server user, or define a new one. Timeout doesn't really play a part in the created server, but does in the call to API.

**Comment by Trdls** [comment]
Changed it so, that we don't set env variables anywhere. Started thinking that the env variable set might not even have worked as I thought it should, so better this way. We now use either user env variables or 300ms in modules and in inventory we use env variables or .ini file configurations.
